### DISCUSSION TITLE
Simplest MSNoise run ever: how to plot dvv

### DISCUSSION THREAD

**Discussion by olivialocke** [ROBelgium/MSNoise#311] [Q&A]
I've been using the developmental version of MSNoise and trying to run the "simplest MSNoise run ever" (section 4.1.1 in the documentation for MSNoise version 1.6).

In the GitHub code, I can find the new commands (msnoise cc ...) for the following steps:
 
msnoise compute_cc 
msnoise stack -r 
msnoise reset STACK
msnoise stack -m
msnoise compute_mwcs
msnoise compute_dtt 

But, I can't figure out how to plot the dvv plot at the end. Using the command ''msnoise cc dvv plot dvv'' I get a ValueError. 

ValueError: Multi-dimensional indexing (e.g. `obj[:, None]`) is no longer supported. Convert to a numpy array before indexing instead.

Do I need to compute dvv first? If so, how? Or, is there a better way to get a dvv plot from a very simple MSNoise run?

**Comment by ThomasLecocq** [comment]
Hey

Yes, msnoise cc dvv compute_dvv should work before plotting. Could be that the doc isn't fully up to date, thanks for pointing it!

Thomas

**Reply by ThomasLecocq** [reply]
http://msnoise.org/doc/master/how_tos.html#run-the-simplest-msnoise-run-ever

**Reply by olivialocke** [reply]
Hi thank you! When I run msnoise cc dvv compute_dvv I get: 

2023-11-14 21:00:58.632793 msnoise [pid 41964][INFO]: *** Starting: Compute DV/V ***
RuntimeWarning: invalid value encountered in scalar divide 
     wstd = np.sqrt(np.sum(w * (d - wavg) ** 2) / ((N - 1) * np.sum(w) / N))

Do you have any suggestions on how to fix this?

**Reply by ThomasLecocq** [reply]
That's just a warning, which could occur when the linear regression fails due to division by zero or else, but should still produce results for all other cases

**Reply by olivialocke** [reply]
Sorry to keep bothering you - I'm still getting the ValueError I originally got and no dvv plot. 

ValueError: Multi-dimensional indexing (e.g. `obj[:, None]`) is no longer supported. Convert to a numpy array before indexing instead.

This is after running the steps for simplest msnoise run ever then ''msnoise cc dvv compute_dvv'' as you suggested and ''msnoise cc dvv plot dvv.''

Have you seen this error before? 

Is there a test case of station data and output that is available? I would like to test see if I can get the same outputs with my set up of MSNoise. 

Thanks again - I appreciate your help!
Olivia

**Reply by ThomasLecocq** [reply]
humm, could be due to a change in pandas... are the DTT/DVV folders created and contain NC files ?

**Reply by olivialocke** [reply]
Yes, the DTT/DVV folders are being created and contain NC files
